src/prune_builds

```diff
@@ -99,20 +131,22 @@ if skip_pruning:
 
 # now delete other build dirs not in the manifest
 
-for build_dir, _ in builds_to_delete:
+for build_dir, _, _ in builds_to_delete:
     shutil.rmtree(os.path.join(builds_dir, build_dir))
 
-
 # and finally prune OSTree repos
-def ostree_prune(repo_name, sudo=False):
-    repo = os.path.join(args.workdir, repo_name)
-    argv = []
-    if sudo:
-        argv.extend(['sudo', '--'])
-    print(f"Pruning {repo_name}")
-    argv.extend(["ostree", "prune", "--repo", repo, "--refs-only",
-                f"--depth={args.keep_last_n-1}"])
-    subprocess.run(argv, check=True)
-
-
-ostree_prune('repo')
+print(f"Pruning repo")
+repo = os.path.join(args.workdir, 'repo')
+
+# For now, we just use the --keep-younger-than CLI here. Doing this more
```

Comment:
before were we pruning based on time, or on number?

Reply:
On number. Though even here, we're still technically pruning on number even though we're using `--keep-younger-than`. It's just an easy way to avoid manually figuring out the depth of the oldest build.